.github/dependabot.yml

```diff
@@ -0,0 +1,39 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: daily
+    commit-message:
+      # Prefix all commit messages with "chore(Dockerfile): "
+      prefix: "chore(Dockerfile): "
+    # Add reviewers
+    reviewers:
```

Comment:
Can you add `14rcole` ([Ryan](https://github.com/orgs/redhat-appstudio/people/14rcole)) to this list?

Reply:
Ack, thanks for catching that!